root/etc/e-smith/templates/httpd/vhost-extra/25IndexesOptions

```diff
@@ -0,0 +1,14 @@
+{
+use esmith::ConfigDB;
+my $vdb = esmith::ConfigDB->open_ro('vhosts') || die("Can't open vhosts db");
+
+my $indexes = $vdb->get_prop("$VhostName",'Indexes') || '';
+return unless ($indexes eq 'enabled');
+
+$OUT .= << 'EOF';
+        #25IndexesOption
+        <LocationMatch "^/">
```

Comment:
<LocationMatch "^/+$">
the root of the virtualhost is browsable, but not the subfolders
<LocationMatch "^/">
the root and the subfolders are browsable

I'm not sure that an option to browse the root and another to browser the root and the subfolder is good. It is complicated